web/src/features/charts/FuturePrice.tsx

```diff
@@ -74,16 +73,27 @@ export function FuturePrice({ futurePrice }: { futurePrice: FuturePriceData | nu
             <ul>
               {Object.entries(filteredPriceData).map(
                 ([date, price]: [string, number]) => (
-                  <li key={date}>
+                  <li
+                    key={date}
+                    className={
+                      isNow(date, granularity)
+                        ? `rounded-md bg-[#18214F]/10 dark:bg-[#848EC0]/10`
```

Comment:
nit: If we're using these hex codes in multiple places, then we can consider giving them more human-readable names in the [tailwind config](https://tailwindcss.com/docs/customizing-colors#naming-your-colors)

Reply:
@Alportan do you think we will use this color again (the purple price bar color)? If we are not planning on using it again right now, I think it's fine to leave it as hex

Reply:
Are these the colours for the price data graph? If yes, then yes, it would be cool to color code certain data that today uses a neutral color even more, such as price and exchanges. I second adding it to the `config` 🙌

Reply:
Yes! I'll add it to the config:)